charts/nginx-ingress-services/templates/secret_federator.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.federator.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: federator-certificate-secret
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: kubernetes.io/tls
+data:
+  ca.crt: {{ .Values.secrets.tlsClientCA | b64enc | quote }}
```

Comment:
This should be optional right? Otherwise how can we tell nginx to use system CA?

Reply:
I don't know if the client CA can be optional. I didn't find a way to make nginx use the system defaults to verify client certificates. That's why I had Let's Encrypt intermediate CA as the default before.